S21/hvkone/ui_txt_conv.py

```diff
@@ -0,0 +1,81 @@
+#---------------------------------
+# DEPENDECIES
+#---------------------------------
+# pip install langdetect
+from langdetect import detect
+
+# pip install translate
+from translate import Translator
+
+# pip install pycountr
+import pycountry
+
+# pip install bcp47
+from bcp47 import bcp47
+
+# pip install langcodes
+# pip install language_data
+from langcodes import *
+
+
+def convert_to_ascii(text):
+    return " ".join(str(ord(char)) for char in text)
+#---------------------------------
+# START OF CODE
+#---------------------------------
+usr_word = input('Type Word (to be converted to lowercase): ')
+
+usr_word_code = usr_word.encode(encoding = 'UTF-8',errors = 'strict')
+print('UTF-8 Input: ', usr_word_code)
+
+usr_asc = convert_to_ascii(usr_word)
+print('ORIGINAL ASCII INPUT: ', usr_asc)
+# Convert to lowercase using ASCII 
+usr_asc_new = ''
+for i in range(len(usr_word)):
+    if(usr_word[i] >= 'A' and usr_word[i] <= 'Z'):
+        usr_asc_new = usr_asc_new + chr((ord(usr_word[i]) + 32))
+    else:
+        usr_asc_new = usr_asc_new + usr_word[i]
+usr_asc_new = convert_to_ascii(usr_asc_new)       
+print('CONVERTED ASCII OUTPUT: ', usr_asc_new)
+
+usr_lang = input('Type Language Code [en]: ')
+usr_regn = input('Type Region Code [US]: ')
+
```

Comment:
It would be beneficial if we read files, instead of giving inputs individually. Because there might be multiple test cases, and it would look time-consuming if we put them individually in the code.

Reply:
Okay, thank you. I was just trying to see if I could input various languages for detection.

Reply:
No problem, that's great we understand each other points.